activestorage/dummy_data.py

```diff
@@ -18,6 +18,13 @@ def _make_data(n=10):
 
     return dd, data
 
+def make_pathological_ncdata(filename='test_pathological_missing.nc', chunksize=(3,3,1), n=10):
```

Comment:
pls rename this to something more descriptive eg `make_ncdata_with_missing_values`

Reply:
Can you please do it, maybe use `make_half_missing_ncdata`?

Reply:
done - replaced "pathological" with "partially_missing"